authors/apps/articles/views.py

```diff
@@ -54,6 +54,14 @@ def get_queryset(self):
                 tag_list__name__icontains=self.request.GET['tag'])
         return articles
 
+    def article_read_time(self, article_body):
```

Comment:
Rename this to ` read_time` and put it below create or as the last method in this class

Reply:
This has been done